internal/Change-DbaSqlServiceStartupMode.ps1

```diff
@@ -1,26 +1,43 @@
 Function Change-DbaSqlServiceStartupMode {
```

Comment:
Actually I guess `Edit` would be the right verb. As well since this is internal we can make it shorter and just say something like `Edit-ServiceStartMode`. It is known/understood that we work with services around SQL Server.